src/libpmemfile-posix/lseek.c

```diff
@@ -202,30 +202,17 @@ pmemfile_lseek_locked(PMEMfilepool *pfp, PMEMfile *file, pmemfile_off_t offset,
 		case PMEMFILE_SEEK_CUR:
 			ret = (pmemfile_off_t)file->offset + offset;
 			if (ret < 0) {
-				if (offset < 0) {
-					new_errno = EINVAL;
-				} else {
-					/*
-					 * From POSIX: EOVERFLOW if
-					 * "...The resulting file offset would
-					 * be a value which cannot be
-					 * represented correctly in an object
-					 * of type off_t..."
-					 */
-					new_errno = EOVERFLOW;
-				}
+				/* Error as in SEEK_SET */
```

Comment:
And please add a little comment about following some existing implementations instead of POSIX, or the lseek man page on linux, so we know why this is so.